examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>I think it'd be great to have a <strong>Sharing Your Model </strong>subsection and move this cell and following 3 cells under it. Ideally this subsection would be just before the <strong>Data Augmentation</strong> subsection</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>